subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -583,9 +568,9 @@ void ticker_worker(void *param)
 	}
 
 	/* Queue the elapsed ticks */
-	if (instance->ticks_elapsed_first == instance->ticks_elapsed_last) {
-		ticker_next_elapsed(&instance->ticks_elapsed_last);
-	}
+	if (instance->ticks_elapsed_first == instance->ticks_elapsed_last)
+		instance->ticks_elapsed_last = INC_MOD(instance->ticks_elapsed_last, DOUBLE_BUFFER_SIZE);
```

Comment:
Enclose within {}, I am think its the convention zephyr-wide

Reply:
I used checkpatch that I know complains if there are curly braces after simple if statements. But you are right, the zephyr documentation requires the use of braces even after simple if statements.